### ISSUE TITLE
Ubuntu Trusty or Debian Stretch? Documentation is in conflict with itself

### ISSUE THREAD

**Issue by HammyHavoc** [LibreTime/libretime#391]
As per http://libretime.org/manual/preparing-the-server/ and http://libretime.org/install/: there's a conflict as to which is the recommendation for a LibreTime server. Some clarification on this in the documentation and in general would be appreciated.

**Comment by hairmare** [comment]
Hi @HammyHavoc 

Welcome to the LibreTime community :)

`/install` is outdated, you should probably go with Debian Trusty or Ubuntu Xenial.

Remember to look at the [release notes](https://github.com/LibreTime/libretime/releases) as well. They have some pointers for getting a successful install.

I'd say we should probably fix this by using a distro and version neutral wording in install. Feel free to open a PR to help clean up the docs if you'd like to.

Cheers,
Lucas

**Comment by HammyHavoc** [comment]
Hey Lucas!

Thanks for the welcome, I'll clean up those docs in the next hour or so for you. :- )

Do you mean Ubuntu Trusty? Debian Trusty isn't a thing (as far as I know).

**Comment by Robbt** [comment]
I'm pretty sure he meant to say Debian Stretch, although there have been some issues with the liquidsoap mp3 support that people have had with Stretch but not with Ubuntu's packages.

**Comment by hairmare** [comment]
Yes, I meant Debian Stretch... The idea being that you grab a recent current LTS so you don't have to upgrade soonish.

**Comment by Power2All** [comment]
Using 14.04 and normal installer script, works fine.
MP3 playback works finally (yay), but the metatagging is bugged hard.
Icecast is not updating with metatags, seems a liquidsoap issue.
